code/datums/helper_datums/teleport.dm

```diff
@@ -229,3 +234,28 @@
 
 	if(do_teleport(affected_mob, pick(L), channel = TELEPORT_CHANNEL_MAGIC, no_effects = TRUE))
 		affected_mob.say("SCYAR NILA [uppertext(thearea.name)]!", forced = "wizarditis teleport")
+
+/obj/effect/temp_visual/teleportation_wake
+	name = "slipspace wake"
+	duration = 30 SECONDS
+	randomdir = FALSE
+	icon = 'icons/effects/effects.dmi'
+	icon_state = null
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	var/turf/destination
+
+/obj/effect/temp_visual/teleportation_wake/Initialize(mapload, turf/destination)
+	// Replace any portals on the current turf
+	for (var/obj/effect/temp_visual/teleportation_wake/conflicting_portal in loc)
+		if (conflicting_portal == src)
+			continue
+		conflicting_portal.destination = destination
+		return INITIALIZE_HINT_QDEL
+	. = ..()
+	src.destination = destination
+	var/image/I = image(icon = 'icons/effects/effects.dmi', icon_state = "bluestream", layer = HIGH_OBJ_LAYER, loc = src)
+	I.appearance_flags = RESET_ALPHA | RESET_COLOR
+	I.alpha = 255
+	// Holy people can see the wakes
+	add_alt_appearance(/datum/atom_hud/alternate_appearance/basic/blessedAware, "slipspace_wake", I)
```

Comment:
imo it doesn't really make sense for blessed people to see the wake? It should be like.. a diag hud thing

Reply:
I like the idea overall but I agree with this, it just feels a little arbitrary for who can see the wakes... Might wanna give it some more thought.

Reply:
probably intended for cult teleports and such, since the chaplain is supposed to directly deal with those.

Reply:
Yea, my idea was that science and magic are somewhat interconnected and that the chaplain can see the cult/magic things for the same reason they can see the wakes.

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/1f88d16d-46dd-4982-b8d3-63f0a6d34188)
Now shows in the diag hud instead